content/security/advisory/2017-07-10.adoc

```diff
@@ -147,10 +147,10 @@ An enumeration of credentials IDs in this plugin now requires the permission to
 If no job context exists, Overall/Administer permission is required.
 
 
-== Unsafe methods in the default whitelist in Script Security Plugin
+== Unsafe methods in the default allowed signature list in Script Security Plugin
```

Comment:
I think "list of approved signatures" is a little better, and is what you used in most other places:

```suggestion
== Unsafe methods in the default list of approved signatures in Script Security Plugin
```